### ISSUE TITLE
ActionView::Template::Error: File to import not found or unreadable: bourbon.

### ISSUE THREAD

**Issue by Overload119** [thoughtbot/bourbon#1024]
I'm having the issue with a Rails 5 app.

Interestingly, I only get the issue when I run the new render method that Rails 5 offers:

```
PostcardsController.render(
        template: 'postcards/preview_back',
        assigns: { postcard: resource },
      )
```

In a Rails console, I see that bourbon isn't in the assets path. Any idea why this would be?

```
[7] pry(main)> Rails.application.config.assets.paths
=> ["/Users/amirsharif/Projects/sapco/app/assets/config",
 "/Users/amirsharif/Projects/sapco/app/assets/fonts",
 "/Users/amirsharif/Projects/sapco/app/assets/images",
 "/Users/amirsharif/Projects/sapco/app/assets/javascripts",
 "/Users/amirsharif/Projects/sapco/app/assets/stylesheets",
 "/Users/amirsharif/Projects/sapco/vendor/assets/javascripts",
 "/Users/amirsharif/Projects/sapco/vendor/assets/stylesheets",
 "/Users/amirsharif/.rvm/gems/ruby-2.4.1/gems/pure-css-rails-0.6.0/vendor/assets/stylesheets",
 "/Users/amirsharif/.rvm/gems/ruby-2.4.1/gems/local_time-1.0.3/app/assets/javascripts",
 "/Users/amirsharif/.rvm/gems/ruby-2.4.1/gems/actioncable-5.0.2/lib/assets/compiled",
 "/Users/amirsharif/Projects/sapco/app/assets/fonts"]
```

My gemfile (I also tried updating to Bourbon 5, no luck). 

```
GEM
  remote: https://rubygems.org/
  specs:
    actioncable (5.1.0)
      actionpack (= 5.1.0)
      nio4r (~> 2.0)
      websocket-driver (~> 0.6.1)
    actionmailer (5.1.0)
      actionpack (= 5.1.0)
      actionview (= 5.1.0)
      activejob (= 5.1.0)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 2.0)
    actionpack (5.1.0)
      actionview (= 5.1.0)
      activesupport (= 5.1.0)
      rack (~> 2.0)
      rack-test (~> 0.6.3)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (5.1.0)
      activesupport (= 5.1.0)
      builder (~> 3.1)
      erubi (~> 1.4)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.3)
    active_record_query_trace (1.5.4)
    active_record_uuid (0.3.1)
      activerecord (>= 3.2)
      mysql2 (~> 0.3.14)
      uuidtools (~> 2.1.2)
    activejob (5.1.0)
      activesupport (= 5.1.0)
      globalid (>= 0.3.6)
    activemodel (5.1.0)
      activesupport (= 5.1.0)
    activerecord (5.1.0)
      activemodel (= 5.1.0)
      activesupport (= 5.1.0)
      arel (~> 8.0)
    activerecord-import (0.18.2)
      activerecord (>= 3.2)
    activesupport (5.1.0)
      concurrent-ruby (~> 1.0, >= 1.0.2)
      i18n (~> 0.7)
      minitest (~> 5.1)
      tzinfo (~> 1.1)
    addressable (2.5.1)
      public_suffix (~> 2.0, >= 2.0.2)
    arel (8.0.0)
    awesome_print (1.7.0)
    aws-sdk (2.9.15)
      aws-sdk-resources (= 2.9.15)
    aws-sdk-core (2.9.15)
      aws-sigv4 (~> 1.0)
      jmespath (~> 1.0)
    aws-sdk-resources (2.9.15)
      aws-sdk-core (= 2.9.15)
    aws-sigv4 (1.0.0)
    bindex (0.5.0)
    bourbon (4.3.4)
      sass (~> 3.4)
      thor (~> 0.19)
    breakpoint (2.7.1)
      sass (~> 3.3)
      sassy-maps (< 1.0.0)
    builder (3.2.3)
    byebug (9.0.6)
    callsite (0.0.11)
    coderay (1.1.1)
    coffee-rails (4.2.1)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0, < 5.2.x)
    coffee-script (2.4.1)
      coffee-script-source
      execjs
    coffee-script-source (1.12.2)
    concurrent-ruby (1.0.5)
    connection_pool (2.2.1)
    domain_name (0.5.20170404)
      unf (>= 0.0.5, < 1.0.0)
    em-websocket (0.5.1)
      eventmachine (>= 0.12.9)
      http_parser.rb (~> 0.6.0)
    encryptor (3.0.0)
    erubi (1.6.0)
    et-orbi (1.0.3)
      tzinfo
    eventmachine (1.2.3)
    execjs (2.7.0)
    faraday (0.12.1)
      multipart-post (>= 1.2, < 3)
    fastimage (2.1.0)
    ffi (1.9.18)
    formatador (0.2.5)
    globalid (0.4.0)
      activesupport (>= 4.2.0)
    gon (6.1.0)
      actionpack (>= 3.0)
      json
      multi_json
      request_store (>= 1.0)
    guard (2.14.1)
      formatador (>= 0.2.4)
      listen (>= 2.7, < 4.0)
      lumberjack (~> 1.0)
      nenv (~> 0.1)
      notiffany (~> 0.0)
      pry (>= 0.9.12)
      shellany (~> 0.0)
      thor (>= 0.18.1)
    guard-compat (1.2.1)
    guard-livereload (2.5.2)
      em-websocket (~> 0.5)
      guard (~> 2.8)
      guard-compat (~> 1.0)
      multi_json (~> 1.8)
    http-cookie (1.0.3)
      domain_name (~> 0.5)
    http_parser.rb (0.6.0)
    i18n (0.8.1)
    invariant (0.2.0)
    jbuilder (2.6.3)
      activesupport (>= 3.0.0, < 5.2)
      multi_json (~> 1.2)
    jmespath (1.3.1)
    js-routes (1.3.3)
      railties (>= 3.2)
      sprockets-rails
    json (2.1.0)
    launchy (2.4.3)
      addressable (~> 2.3)
    letter_opener (1.4.1)
      launchy (~> 2.2)
    libv8 (3.16.14.19)
    listen (3.0.8)
      rb-fsevent (~> 0.9, >= 0.9.4)
      rb-inotify (~> 0.9, >= 0.9.7)
    lob (3.0.0)
      rest-client (>= 1.8, < 3.0)
    local_time (1.0.3)
      coffee-rails
    loofah (2.0.3)
      nokogiri (>= 1.5.9)
    lumberjack (1.0.11)
    mail (2.6.5)
      mime-types (>= 1.16, < 4)
    meta_request (0.4.2)
      callsite (~> 0.0, >= 0.0.11)
      rack-contrib (~> 1.1)
      railties (>= 3.0.0, < 5.2.0)
    method_source (0.8.2)
    mime-types (3.1)
      mime-types-data (~> 3.2015)
    mime-types-data (3.2016.0521)
    mina (1.0.6)
      open4 (~> 1.3.4)
      rake
    mina-nginx (0.0.3)
      mina (~> 1)
    mina-puma (1.0.1)
      mina (~> 1.0.0)
      puma (>= 2.13)
    mina-rsync (1.1.0)
      mina (>= 0.3.0, < 2)
    mina-sidekiq (1.0.1)
      mina (>= 1.0.2)
    mini_magick (4.7.0)
    mini_portile2 (2.1.0)
    minitest (5.10.1)
    multi_json (1.12.1)
    multipart-post (2.0.0)
    mustermann (1.0.0)
    mysql2 (0.3.21)
    nenv (0.3.0)
    netrc (0.11.0)
    nio4r (2.0.0)
    nokogiri (1.7.1)
      mini_portile2 (~> 2.1.0)
    notiffany (0.1.1)
      nenv (~> 0.1)
      shellany (~> 0.0)
    oj (3.0.5)
    open4 (1.3.4)
    pry (0.10.4)
      coderay (~> 1.1.0)
      method_source (~> 0.8.1)
      slop (~> 3.4)
    pry-rails (0.3.6)
      pry (>= 0.10.4)
    public_suffix (2.0.5)
    puma (3.8.2)
    pure-css-rails (0.6.0)
    rack (2.0.1)
    rack-contrib (1.2.0)
      rack (>= 0.9.1)
    rack-livereload (0.3.16)
      rack
    rack-protection (2.0.0)
      rack
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (5.1.0)
      actioncable (= 5.1.0)
      actionmailer (= 5.1.0)
      actionpack (= 5.1.0)
      actionview (= 5.1.0)
      activejob (= 5.1.0)
      activemodel (= 5.1.0)
      activerecord (= 5.1.0)
      activesupport (= 5.1.0)
      bundler (>= 1.3.0, < 2.0)
      railties (= 5.1.0)
      sprockets-rails (>= 2.0.0)
    rails-dom-testing (2.0.2)
      activesupport (>= 4.2.0, < 6.0)
      nokogiri (~> 1.6)
    rails-html-sanitizer (1.0.3)
      loofah (~> 2.0)
    rails_param (0.10.0)
    railties (5.1.0)
      actionpack (= 5.1.0)
      activesupport (= 5.1.0)
      method_source
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (12.0.0)
    rb-fsevent (0.9.8)
    rb-inotify (0.9.8)
      ffi (>= 0.5.0)
    redis (3.3.3)
    redis-namespace (1.5.3)
      redis (~> 3.0, >= 3.0.4)
    ref (2.0.0)
    request_store (1.3.2)
    rest-client (2.0.2)
      http-cookie (>= 1.0.2, < 2.0)
      mime-types (>= 1.16, < 4.0)
      netrc (~> 0.8)
    ruby_postal (1.0.0)
    rufus-scheduler (3.4.0)
      et-orbi (~> 1.0)
    sass (3.4.23)
    sass-rails (5.0.6)
      railties (>= 4.0.0, < 6)
      sass (~> 3.1)
      sprockets (>= 2.8, < 4.0)
      sprockets-rails (>= 2.0, < 4.0)
      tilt (>= 1.1, < 3)
    sassy-maps (0.4.0)
      sass (~> 3.3)
    shellany (0.0.1)
    sidekiq (5.0.0)
      concurrent-ruby (~> 1.0)
      connection_pool (~> 2.2, >= 2.2.0)
      rack-protection (>= 1.5.0)
      redis (~> 3.3, >= 3.3.3)
    sidekiq-scheduler (2.1.4)
      redis (~> 3)
      rufus-scheduler (~> 3.2)
      sidekiq (>= 3)
      tilt (>= 1.4.0)
    sinatra (2.0.0)
      mustermann (~> 1.0)
      rack (~> 2.0)
      rack-protection (= 2.0.0)
      tilt (~> 2.0)
    slim (3.0.8)
      temple (>= 0.7.6, < 0.9)
      tilt (>= 1.3.3, < 2.1)
    slim-rails (3.1.2)
      actionpack (>= 3.1)
      railties (>= 3.1)
      slim (~> 3.0)
    slop (3.6.0)
    spring (2.0.1)
      activesupport (>= 4.2)
    spring-watcher-listen (2.0.1)
      listen (>= 2.7, < 4.0)
      spring (>= 1.2, < 3.0)
    sprockets (3.7.1)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.2.0)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    stripe (2.8.0)
      faraday (~> 0.9)
    temple (0.8.0)
    therubyracer (0.12.3)
      libv8 (~> 3.16.14.15)
      ref
    thor (0.19.4)
    thread_safe (0.3.6)
    tilt (2.0.7)
    tzinfo (1.2.3)
      thread_safe (~> 0.1)
    uglifier (3.2.0)
      execjs (>= 0.3.0, < 3)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.7.4)
    uuidtools (2.1.5)
    web-console (3.5.0)
      actionview (>= 5.0)
      activemodel (>= 5.0)
      bindex (>= 0.4.0)
      railties (>= 5.0)
    webpacker (1.2)
      activesupport (>= 4.2)
      multi_json (~> 1.2)
      railties (>= 4.2)
    webpacker-react (0.2.0)
      webpacker
    websocket-driver (0.6.5)
      websocket-extensions (>= 0.1.0)
    websocket-extensions (0.1.2)

PLATFORMS
  ruby

DEPENDENCIES
  active_record_query_trace
  active_record_uuid
  activerecord-import
  awesome_print
  aws-sdk
  bourbon
  breakpoint
  byebug
  encryptor
  fastimage
  gon
  guard
  guard-livereload
  invariant
  jbuilder
  js-routes
  letter_opener
  listen (~> 3.0.5)
  lob
  local_time
  meta_request
  mina
  mina-nginx
  mina-puma
  mina-rsync
  mina-sidekiq
  mini_magick
  mysql2
  oj
  pry-rails
  puma (~> 3.0)
  pure-css-rails
  rack-livereload
  rails (~> 5.1)
  rails_param
  redis
  redis-namespace
  ruby_postal
  sass-rails
  sidekiq
  sidekiq-scheduler (~> 2.0)
  sinatra
  slim
  slim-rails
  spring
  spring-watcher-listen (~> 2.0.0)
  stripe
  therubyracer
  uglifier (>= 1.3.0)
  web-console (>= 3.3.0)
  webpacker
  webpacker-react

BUNDLED WITH
   1.14.6
```

**Comment by tysongach** [comment]
We just published [v5.0.0.beta.8](https://github.com/thoughtbot/bourbon/releases/tag/v5.0.0.beta.8) which potentially solves this issue. Can you try upgrading and let us know if the problem persists or gets resolved?

**Comment by tysongach** [comment]
Closing due to lack of response. Please re-open if we can help in any way!
